packages/ckeditor5-engine/src/controller/datacontroller.js

```diff
@@ -250,7 +250,7 @@ export default class DataController {
 		// Convert markers.
 		// For document fragment, simply take the markers assigned to this document fragment.
 		// For model root, all markers in that root will be taken.
-		// For model element, we need to check which markers are intersecting with this element and relatively modify the markers' ranges.
+		// For model element, we need to check which markers are contained in this element and relatively modify the markers' ranges.
```

Comment:
This comment change is not entirely true - markers that are partly in that element are also included.